spec/features/saml_authentication_spec.rb

```diff
@@ -156,24 +157,7 @@
       create_user("you@example.com")
 
       visit 'http://localhost:8020/users/saml/sign_in/?entity_id=http%3A%2F%2Flocalhost%3A8020%2Fsaml%2Fmetadata'
-      expect(current_url).to match(%r(\Ahttp://www.example.com/sso\?SAMLRequest=))
-    end
-
-    it "logs a user out of the IdP via the SP" do
```

Comment:
This test could no longer work. Capybara now follows redirects, so it would end up back at the sign-in page, which automatically tried to connect to `localhost:8009` because the root path includes no entity ID.